### PR TITLE
Add requiresOptin

### PR DESCRIPTION
**Pull request by warmkesselj**
### Summary of changes

Added an annotation for an experimental API to Credit Messaging.

### Checklist

 - [ ] Added a changelog entry
 - [ ] Relevant test coverage

### Authors
@jwarmkessel 
@KunJeongPark 
@saperi22

### CODE REVIEW

**Comment by jaxdesmarais** [comment]
I looks like we have some failing tests and the linter is failing

File: Demo/src/main/java/com/braintreepayments/demo/PayPalMessagingFragment.kt [hunk 1]
```diff
@@ -22,6 +24,7 @@ class PayPalMessagingFragment : BaseFragment(), PayPalMessagingListener {
         return inflater.inflate(R.layout.fragment_paypal_messaging, container, false)
     }
 
+    @OptIn(ExperimentalBetaApi::class)
```
Comment (sarahkoop) [inline Demo/src/main/java/com/braintreepayments/demo/PayPalMessagingFragment.kt:27]: Just curious - is this required if we've already opted the entire fragment class in?

Comment (warmkesselj) [reply Demo/src/main/java/com/braintreepayments/demo/PayPalMessagingFragment.kt:27]: Looks like it's not required. Removing it.

**Review by sarahkoop — APPROVED** [review]

**Review by agedd — APPROVED** [review]
🚀

**Pull request merged**